### ISSUE TITLE
Update the binding

### ISSUE THREAD

**Issue by Yassa-hue** [shakacode/rescript-react-on-rails#6]
Update the binding to match the changes of ROR API.

**Comment by LewWadoo** [comment]
@Yassa-hue, are you working on the issue?

**Comment by Yassa-hue** [comment]
> @Yassa-hue, are you working on the issue?

No, not anymore. it is related to the [Add Rescript page to the RWRT project PR](https://github.com/shakacode/react-webpack-rails-tutorial/pull/552).
I don't think the problem is this binding. I think the problem is some issue on the RWRT project and may @ahangarha or @alex35mil  correct me if I'm mistaken.
